File: haskell/exercism/LuciansLusciousLasagna.hs
module LuciansLusciousLasagna (elapsedTimeInMinutes, expectedMinutesInOven, preparationTimeInMinutes) where

expectedMinutesInOven = 40

preparationTimeInMinutes layers = layers * 2

elapsedTimeInMinutes layers timeCooked = preparationTimeInMinutes layers + timeCooked

File: fsharp/exercism/LuciansLusciousLasagna.fs
module LuciansLusciousLasagna

let expectedMinutesInOven = 40

let remainingMinutesInOven timeSoFar = expectedMinutesInOven - timeSoFar

let preparationTimeInMinutes layers = layers * 2

let elapsedTimeInMinutes layers timeSoFar =
    preparationTimeInMinutes layers + timeSoFar
